packages/components/src/border-control/border-control/README.md

```diff
@@ -58,49 +54,49 @@ colors are organized by multiple origins.
 
 Each color may be an object containing a `name` and `color` value.
 
-- Required: No
-- Default: `[]`
+-   Required: No
+-   Default: `[]`
 
 ### `disableCustomColors`: `boolean`
 
 This toggles the ability to choose custom colors.
 
-- Required: No
+-   Required: No
 
 ### `disableUnits`: `boolean`
 
 This controls whether unit selection should be disabled.
 
-- Required: No
+-   Required: No
 
 ### `enableAlpha`: `boolean`
 
 This controls whether the alpha channel will be offered when selecting
 custom colors.
 
-- Required: No
-- Default: `false`
+-   Required: No
+-   Default: `false`
```

Comment:
Shouldn't this be `true` actually?

https://github.com/WordPress/gutenberg/blob/72436e1f365a5c099285c92e750bcb3adbbb3a4c/packages/components/src/border-control/border-control/hook.ts#L33

Reply:
Fixed. I'd like to try generating these automatically in the future so that they're kept in sync with the actual types and the JSDocs, including defaults.

Reply:
Absolutely. It was a surprise to find this there in the first place.

Reply:
> I'd like to try generating these automatically in the future so that they're kept in sync with the actual types and the JSDocs, including defaults.

Yup, I believe that's part of Lena's future planned work